scanners/zap/zap.py

```diff
@@ -151,7 +151,7 @@ def get_update_command(self):
             *self._get_standard_options(),
             "-cmd",
         ]
-        if self.my_conf("miscOptions.updateAddons", default=True):
+        if self.my_conf("miscOptions.updateAddons"):
```

Comment:
yes, in zap_podman, the situation is a little dirty because, afaik, podman can run only a single command, and since we need to run the update separately from the scan, the code needs to cheat a little with a "sh -c '<update>; <scan>'". 
When there's no need to handle plugins, it's would probably be preferable to avoid this `sh -c` all together, especially if updateAddons is off by default.

I will improve on that, such that and skip the `handle_plugin` when possible

Reply:
so, I tried to improve this a bit, but: `_handle_plugins()` can't actually do the same as its zap_none counterpart, because it can't actually run a podman command.

I updated in this way:
- `_handle_plugins()`  now simply returns the command to handle the plugins as a list (i.e.: empty list if there's nothing to do)
- the caller (`run()`) will then either run the regular ZAP command if there are not plugin to be handled (now default), or bundle both the plugin command and the scan command in a single `sh` wrapper.

Let me know if that's a better solution, or if we should approach it differently.

Reply:
This approach looks like a good improvement. 

Just to understand better, why are both commands run in a single `podman` execution? Could they run separately in different `podman` executions while sharing storage? I’m not suggesting a change, just curious about how it works

Reply:
It should be doable, yes